README.md

```diff
@@ -1,26 +1,74 @@
+<p align="center"><a href="https://azure.microsoft.com/en-us/services/notification-hubs/"><img src="./nh-logo.svg?sanitize=true"/></a></p>
 
-## Getting Started with Android SDK
-Please follow our [tutorial](https://docs.microsoft.com/en-us/azure/notification-hubs/notification-hubs-android-push-notification-google-fcm-get-started) to start using the Android SDK with Microsoft Azure Notification Hubs.
+# Azure Notification Hubs - Android SDK
 
-## Download the source code
-To get the source code of our SDKs and samples via **git** just type:
+Azure Notification Hubs provides a multi-platform, scaled-out push infrastructure that enables you to send mobile push notifications from any backend (in the cloud or on-premises) to any mobile platform. To learn more, visit our [Developer Center](https://azure.microsoft.com/en-us/documentation/services/notification-hubs).
 
-```bash
-git clone https://github.com/Azure/azure-notificationhubs-android.git
-cd azure-notificationhubs-android
+
+## Getting Started with ANH Android SDK
+
+### Reference with Gradle
+
+This library is published on [JFrog Bintray](https://bintray.com/microsoftazuremobile/SDK/Notification-Hubs-Android-SDK#files/com/microsoft/azure/notification-hubs-android-sdk). [Once you've completed steps 1 through 3 here to setup your app with Firebase](https://firebase.google.com/docs/android/setup#console), adding a reference to this project is as simple as editting two files in your project:
+
+_{project-root}/build.gradle:_
+
+``` groovy
+// This is not a complete build.gradle file, it only highlights the portions you'll need to use ANH.
+
+allprojects {
+    repositories {
+        // Ensure you have the following repsoitory in your "allprojects", "repositories" section.
+        maven {
+            url 'https://dl.bintray.com/microsoftazuremobile/SDK'
+        }
+    }
+}
+```
+
+_{project-root}/{your-module}/build.gradle:_
+``` groovy
+// This is not a complete build.gradle file, it only highlights the portions you'll need to use ANH.
+
+dependencies {
+    // Ensure the following line is included in your app/library's "dependencies" section.
+    implementation 'com.microsoft.azure:notification-hubs-android-sdk:v1.0.0-preview3'
+}
 ```
 
-## Building and Referencing the SDK
+## Repository Contents
+
+### [`com.microsoft.windowsazure.messaging.notificationhubs`](./notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs)
+The code found in this namespace uses Azure Notification Hub's [Installation flow](https://docs.microsoft.com/en-us/azure/notification-hubs/notification-hubs-push-notification-registration-management#installations). It is heavily influenced by App Center Push's SDK. It was designed to do more than just act as a REST client, instead abstracting away your interactions with ANH. New projects that are looking to use Azure Notification Hub should use the constructs found in this namespace.
```

Comment:
I think we should be consistent on if we want to use `ANH` or `Azure Notification Hub(s)` throughout the documentation 😆

Reply:
If we do decide to use `ANH`, it would be helpful to have `ANH (Azure Notification Hubs)` somewhere in the docs just to declare what the acronym is~

Reply:
Fixed in 7c4c1aa70cbd50669cee2a8f962aa8f9bdbc475f